client/src/locales/fi.json

```diff
@@ -231,6 +231,7 @@
     "signup": "Ilmoittaudu",
     "cannotSignupMoreGames": "Voit ilmoittautua vain kolmeen peliin per alkamisaika",
     "gamePriority": "Peli on toiveissani sijalla",
-    "confirm": "Vahvista"
+    "confirm": "Vahvista",
+    "cannotSignupMoreThanOneGame": "Olet päässyt peliin {{GAME}}. Et voi ilmoittautua muihin samaan aikaan oleviin peleihin."
```

Comment:
Mietin tätä sanamuotoa `Olet päässyt peliin`, kun `päässä peliin` toimii algoritmi-sijoituksessa, mutta tässä kohtaa voisi toimia paremmin `Olet ilmoittautunut peliin`.

Reply:
Sama englanniksi: `You already entered` -> `You already signed`